release.yaml

```diff
@@ -1,20 +1,20 @@
-rancher-eks-operator:
-  - 100.0.4+up1.1.5-rc1
-rancher-eks-operator-crd:
-  - 100.0.4+up1.1.5-rc1
 rancher-aks-operator:
```

Comment:
This commit just re-orders everything alphabetically. Doesn't change what has been committed (indicated by CI still passing).